src/test/java/spoon/test/template/TemplateTest.java

```diff
@@ -297,6 +306,10 @@ public void testTemplateMatcher() throws Exception {
 			CtMethod meth = (CtMethod) templateKlass.getElements(new NameFilter("matcher4")).get(0);
 
 			// together with the appropriate @Parameter f, this means
+/*
+ * this is not correct. See the new test below.
```

Comment:
could you update the contract then so that we see the old version of the contract deleted and the version added?